static/js/attachments_ui.js

```diff
@@ -33,9 +19,45 @@ function bytes_to_size(bytes, kb_with_1024_bytes) {
         size = Math.round(bytes / Math.pow(kb_size, i) * 10) / 10;
     }
     return size + ' ' + sizes[i];
+};
+
+exports.percentage_used_space = function (uploads_size) {
+    if (page_params.realm_upload_quota === null) {
+        return null;
+    }
+    return (uploads_size / page_params.realm_upload_quota).toFixed(1);
```

Comment:
This is missing a `100 *` to get percentage, and thus always outputs 0.0%.  I found this by testing uploading a 3MB file after setting the quota to 1GB.  You should definitely has tested this manually. :(

Reply:
@timabbott Sorry. I actually tested this a lot of times. I was thinking it was showing 0% due to the small upload space used. So I was returning just `uploads_size` during the testing to make sure that upload_size is getting updated during events.

Reply:
Attaching a screenshot I did during the testing phase to make sure that the percentage changes when I upload a lot of big files. It should have shown 10% instead of 0.1% but i did this.
https://chat.zulip.org/user_uploads/2/dd/FHcAP7QFoHyqdnG890EcTk-6/Screenshot-from-2019-01-15-13-06-57.png

Reply:
Attaching a screenshot I took during the testing phase to make sure that the percentage changes when I upload a lot of big files. It should have shown 10% instead of 0.1% but i did test this a lot. I was not super careful with math. But I did test this a lot. 
https://chat.zulip.org/user_uploads/2/dd/FHcAP7QFoHyqdnG890EcTk-6/Screenshot-from-2019-01-15-13-06-57.png

Reply:
Attaching a screenshot I took during the testing phase to make sure that the percentage changes when I upload a lot of big files. It should have shown 10% instead of 0.1% but i did test this a lot. I was not careful with math which was a mistake.
https://chat.zulip.org/user_uploads/2/dd/FHcAP7QFoHyqdnG890EcTk-6/Screenshot-from-2019-01-15-13-06-57.png